scripts/pack-missing-tests.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+if [ -n "${1}" ]; then
+    DIR=${1}
+else
+    echo "Need a pack dir"
+    exit 1
+fi
+
+ACTIONS=$(cd ${DIR}/actions;ls -1 *.py)
```

Comment:
I've changed that to PACKS, which is hopefully more correct. I had both PACK and DIR at one point. However that did not work when it came to adding the Make target.